config/routes.rb

```diff
@@ -95,6 +95,8 @@
 
     get "/collections/:slug", to: "document_collection#show", as: :document_collection
 
+    get "/consultations/:slug(.:locale)", to: "consultation#show"
```

Comment:
Are consultations localisable? (if not, we don't need the .locale)

Reply:
Yep they are e.g. https://www.gov.uk/government/consultations/ymgynghoriad-ar-gyflwyno-ffioedd-yswiriant-a-ganiateir-ar-gyfer-landlordiaid-rhydd-ddeiliaid-ac-asiantiaid-rheoli-eiddo.cy